gnucash/register/ledger-core/split-register-copy-ops.c

```diff
@@ -427,3 +436,24 @@ void gnc_float_txn_to_txn_swap_accounts (const FloatingTxn *ft, Transaction *txn
     if (do_commit)
         xaccTransCommitEdit (txn);
 }
+
+void gnc_float_txn_free (FloatingTxn *ft)
+{
+    GList *iter;
+
+    g_return_if_fail (ft);
+
+    CACHE_REMOVE (ft->m_num);
+    CACHE_REMOVE (ft->m_description);
+    CACHE_REMOVE (ft->m_notes);
+    CACHE_REMOVE (ft->m_doclink);
+
+    for (iter = ft->m_splits; iter; iter = iter->next)
+    {
+        FloatingSplit *fs = iter->data;
+        gnc_float_split_free (fs);
+    }
+    g_list_free (ft->m_splits);
```

Comment:
You can use `g_list_free_full (ft->m_splits, (GDestroyNotify)gnc_float_split_free);` instead of writing the loop yourself.